### PR TITLE
sylius 1.12 support

### PR DESCRIPTION
**Pull request by clairecoloma**
Update for Sylius 1.12 :)

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -14,23 +14,24 @@ jobs:
...
     env:
       SYMFONY_ARGS: --no-tls
       COMPOSER_ARGS: --prefer-dist
       DOCKER_INTERACTIVE_ARGS: -t
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+        - uses: actions/checkout@v3
+        - uses: actions/setup-node@v3
```
Comment (jacquesbh) [inline .github/workflows/tests.yaml:26]:
```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
```

File: .php-version.dist [hunk 2]
```diff
@@ -1 +1,2 @@
 8.0
```
Comment (jacquesbh) [inline .php-version.dist:1]:
```suggestion
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]
Just a few changes at first. Let's see how the tests react.

Also I'll run the project locally to give it a try in 1.12!

**Comment by clairecoloma** [comment]
Changes done :D

**Review by maximehuran — APPROVED** [review]

**Review by delyriand — APPROVED** [review]

**Comment by delyriand** [comment]
Thanks, @clairecoloma!

**Pull request merged**